### PR TITLE
Fix gradle

### PR DESCRIPTION
**Pull request by kevinkowa**
I'm using https://central.sonatype.org/publish/publish-gradle/, the spring-boot-starter, and the previous build we had as a reference on how to do this

### CODE REVIEW

File: .utility/bintray-release.gradle [hunk 1]
```diff
@@ -1,98 +1,72 @@
...
+                        id 'kevin'
+                        name 'Kevin Kowalski'
+                        email 'kevin.kowalski@ibm.com'
+                    }
+                    developer {
+                        id 'german'
+                        name 'German Attanasio'
+                        email 'germanatt@us.ibm.com'
+                    }
+                    developer {
```
Comment (repjarms) [inline .utility/bintray-release.gradle:63]: We should remove Logan

**Review by repjarms — APPROVED** [review]
Looks good to me. Just need to remove Logan as he is no longer an author.

File: .utility/bintray-release.gradle (hunk 1 above)
Comment (kevinkowa) [reply .utility/bintray-release.gradle:63]: alright!

**Review by germanattanasio — APPROVED** [review]
LGTM

File: .utility/bintray-release.gradle [hunk 2]
```diff
@@ -1,98 +1,67 @@
...
-                def root = asNode()
-                root.appendNode('description', libraryDescription)
-                root.appendNode('name', libraryName)
-                root.appendNode('url', 'https://cloud.ibm.com/developer/watson/dashboard')
-                root.children().last() + pomConfig
+                developers {
+                    developer {
+                        id 'kevin'
+                        name 'Kevin Kowalski'
+                        email 'kevin.kowalski@ibm.com'
```
Comment (mediumTaj) [inline .utility/bintray-release.gradle:56]: any way we can make these functional id?

**Review by mediumTaj — APPROVED** [review]
One question but if we can't no problem

**Review by mediumTaj — APPROVED** [review]

**Pull request merged**